lib/takeaway_menu.rb

```diff
@@ -0,0 +1,58 @@
+require_relative 'menu'
+require_relative 'text'
+
+class Takeaway
+  attr_accessor :total
```

Comment:
Do we want to be able to alter this from the command line? is that what accessor does? 
Or do we just want to be able to alter it from within the programme?